nix/launcher-config.nix

```diff
@@ -34,6 +34,7 @@ let
   configDir = configFilesSource: {
     linux = configFilesSource;
     macos64 = if devShell then configFilesSource else "\${DAEDALUS_INSTALL_DIRECTORY}/../Resources";
+    macos64-arm = if devShell then configFilesSource else "\${DAEDALUS_INSTALL_DIRECTORY}/../Resources";
```

Comment:
I would very much like to turn all those instances in our source code to standard arch+os names like `aarch64-darwin`. Why complicate?